.golangci.yml

```diff
@@ -71,7 +71,7 @@ linters-settings:
       - name: argument-limit
         severity: warning
         disabled: false
-        arguments: [ 9 ]
+        arguments: [ 12 ]
```

Comment:
OMG ;-)

Reply:
we can split that identity service into two in order to lower down that number